libs/payments/ui/src/lib/client/components/CheckoutForm/index.tsx

```diff
@@ -158,8 +158,7 @@ export function CheckoutForm({
       displayName: fullName,
     });
 
-    // TODO - To be added in M3B - Redirect customer to '/processing' page
-    router.push('./start');
+    router.push('./processing');
     // TODO - To be moved in M3B - Confirm Payment on '/processing' page
```

Comment:
can these comments be removed too? They're for a slightly different flow, but it looks like the redirects flow needs to be explicitly turned on in our stripe dashboard

https://docs.stripe.com/testing#redirects

Reply:
Comments can be removed. These are legacy and are no longer applicable.